app/jobs/daily_maintenance_job.rb

```diff
@@ -0,0 +1,8 @@
+class DailyMaintenanceJob < ApplicationJob
+  queue_as :default
+
+  def perform(*args)
+    ReadRibbon.expire_old_ribbons!
+    CommentStat.daily_fill!
```

Comment:
Please split this into two named jobs. The overhead of setting up a cron job led to putting these in the same script, but they'll be better off independent of each other.